docs/ko/docs/extra-data-types.md

```diff
@@ -0,0 +1,65 @@
+# 추가 데이터 타입들
+
+지금까지, 일반적인 데이터 타입을 사용했습니다. 예를 들어:
```

Comment:
```suggestion
지금까지, 일반적인 데이터 자료형을 사용했습니다. 예를 들어 아래와 같습니다:
```

> use less English as you can.

Check #2017 and it would be better to complete the sentence for natural.